### DISCUSSION TITLE
Improve Album views

### DISCUSSION THREAD

**Discussion by KillerParticles** [ciderapp/Cider#691] [Added Features]
Add various stuff from AM's Album view to Cider, stuff like the Genre and Year released, the Album description (apparently bugged), star ratings for the most popular songs and the add to library button on the song card.

Top is AM web, bottom is Cider:
![image](https://user-images.githubusercontent.com/93286201/159956203-4295133c-b6d7-4a30-bc65-e62fd99230a0.png)
![image](https://user-images.githubusercontent.com/93286201/159956077-c4057593-fdf0-4eb1-b39f-5d008acc31a7.png)

**Comment by Amaru8** [comment]
Implemented a long time ago, locking discussion.
